Starcounter/Blending
# Problem 241: attaching rules page improvements

**Problem #241:**
After updated `Blender` was merged into `level1` and Blending apps were adopted to the new `Starcounter.Advanced` namespace we can start using new features and signatures of the `Blender` class API methods.

**Solution:**
1. Was added use the `BlendingRuleAdded` event.
2. Were rewritten invocations of the Activate/Deactivate methods.

**Note:**
@alemoi clarified that state of the rules is not preserved in the `Blender` inner dictionary between app restart: https://github.com/Starcounter/Blending/pull/250#issuecomment-405658503.

**Test:**
✅ Manual check steps:
1. Run `Blending.Mapper`
2. Run `SignIn.Mapper`
3. Run `UserAdmin.Mapper`

**Screenshots:**
![creation_attachment_rules_using_event_1](https://user-images.githubusercontent.com/17431807/42864820-61e48108-8a70-11e8-9132-25d30c90dbfc.PNG)
![creation_attachment_rules_using_event_2](https://user-images.githubusercontent.com/17431807/42864821-62089a02-8a70-11e8-8495-f102fd9cfeab.PNG)
